### PR TITLE
Meta

### PR DESCRIPTION
**Pull request by tpreusse**
screen shot 2017-09-07 at 11 44 12

screen shot 2017-09-07 at 11 42 53

### CODE REVIEW

File: components/editor/modules/meta/FBPreview.js [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+  fbDescription: css({
+    fontFamily: 'sans-serif',
+    fontSize: 12,
+    lineHeight: '16px',
+    maxHeight: 80,
+    overflow: 'hidden'
+  }),
+  fbDomain: css({
+    fontFamily: 'sans-serif',
+    fontSize: 11,
```
Comment (uxengine) [inline components/editor/modules/meta/FBPreview.js]: Just a question:
Why "fontSize: 11" but "lineHeight: '11px'" (including "px")? Could you also just do "lineHeight: 11", or will glamor complain?

File: components/editor/modules/meta/ui.js [hunk 2]
```diff
@@ -0,0 +1,166 @@
...
+  const node = state.document
+
+  const genericKeys = Set([
+    'slug',
+    'emailSubject',
+    'title',
+    'image',
+    'description'
+  ])
+  const gernericDefaultValues = Map(genericKeys.map(key => [key, '']))
```
Comment (uxengine) [inline components/editor/modules/meta/ui.js]: Typo: s/gerneric/generic

File: components/editor/modules/meta/ui.js [hunk 3]
```diff
@@ -0,0 +1,166 @@
...
+    paddingTop: '7px !important',
+    paddingBottom: '6px !important'
+  })
+}
+
+const Form = ({
+  t,
+  onInputChange,
+  data,
+  getWidth = key => key.match(/title/i) ? '100%' : ''
```
Comment (uxengine) [inline components/editor/modules/meta/ui.js]: I know you love inlining, but I would move that out into a separate
const getWidth = key => key.match(/title/i) ? '100%' : ''

File: lib/serializer/meta.js [hunk 4]
```diff
@@ -0,0 +1,37 @@
...
+
+    let meta = {}
+    if (yamlNode) {
+      meta = jsYaml.safeLoad(yamlNode.value)
+      parent.children = parent.children.filter(node => node !== yamlNode)
+    }
+    parent.meta = meta
+  }
+
+  return transform
```
Comment (uxengine) [inline lib/serializer/meta.js]: Why not just return the arrow function directly?
return (parent) => {
...
}

File: lib/serializer/meta.test.js [hunk 5]
```diff
@@ -0,0 +1,40 @@
...
+title: Test
+---
+
+# A Test
+`
+  const state = serializer.deserialize(md)
+  const node = state.document
+
+  assert.equal(node.data.get('title'), 'Test')
+  assert.equal(node.data.get('test'), true)
```
Comment (uxengine) [inline lib/serializer/meta.test.js]: Just for readability reasons, I would probably not use a similar value ('Test') and property key ('test') here. But maybe I'm just old ;)

So something like
assert.equal(node.data.get('title'), 'Some title')
assert.equal(node.data.get('foo'), true)

File: components/editor/modules/meta/FBPreview.js (hunk 1 above)
Comment (tpreusse) [reply components/editor/modules/meta/FBPreview.js]: `line-height` accepts unitless number – interpreted as multipler of `font-size`. So glamor would translate 11 to unitless 11 (11 * 11 = 121px)—when css properties don't accept unitless glamor automatically adds `px`—`fontSize: 11` becomes `font-size: 11px`

**Pull request merged**